packages/cli-plugin-scaffold-react-component/template/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "[PACKAGE_NAME]",
+  "version": "1.0.0",
+  "description": "A simple React component that can be imported into the application.",
+  "license": "MIT",
+  "scripts": {
+    "build": "rimraf ./dist '*.tsbuildinfo' && babel src -d dist --source-maps --copy-files --extensions \".ts,.tsx\" && yarn postbuild",
+    "watch": "babel src -d dist --source-maps --copy-files --extensions \".ts,.tsx\" --watch",
+    "postbuild": "cp package.json dist/ && tsc -p tsconfig.build.json"
+  },
+  "dependencies": {
+    "react": "^16.14.0"
+  },
+  "devDependencies": {
```

Comment:
Check dev dependencies here. For example, I can see `"@babel/preset-react"` missing.